drivers/kodi/driver.js

```diff
@@ -4,6 +4,12 @@
 var KodiWs = require('node-kodi-ws')
 var Fuse = require('fuse.js')
 var Utils = require('../../libs/utils')
+const http = require('http');

```

Comment:
I need to use http since the most "GUI.ActivateWindow" json commands lock up the TCP stack.
I can't find any solution to solve this with RPC.